solutions/security/cloud/asset-disc-aws.md

```diff
@@ -0,0 +1,319 @@
+
+* The following trust policy:
+
+::::{dropdown} Click to expand policy
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<Management Account ID>:root"
+            },
+            "Action": "sts:AssumeRole"
+        },
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "ec2.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+
+::::
+
+
+* The AWS-managed `SecurityAudit` policy.
+
+::::{important}
+You must replace `<Management account ID>` in the trust policy with your AWS account ID.
+::::
+
+
+* Next, for each account you want to scan in the organization, create an IAM role named `cloudbeat-securityaudit` with the following policies:
+
+    * The AWS-managed `SecurityAudit` policy.
+    * The following trust policy:
+
+
+::::{dropdown} Click to expand policy
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<Management Account ID>:role/cloudbeat-root"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+
+::::
+
+
+::::{important}
+You must replace `<Management account ID>` in the trust policy with your AWS account ID.
+::::
+
+
+After creating the necessary roles, authenticate using one of the manual authentication methods.
+
+::::{important}
+When deploying to an organization using any of the authentication methods below, you need to make sure that the credentials you provide grant permission to assume `cloudbeat-root` privileges.
+::::
+
+
+
+### Manual authentication methods [cad-aws-set-up-manual]
+
+* [Default instance role (recommended)](/solutions/security/cloud/asset-disc-aws.md#cad-aws-use-instance-role)
+* [Direct access keys](/solutions/security/cloud/asset-disc-aws.md#cad-aws-use-keys-directly)
+* [Temporary security credentials](/solutions/security/cloud/asset-disc-aws.md#cad-aws-temp-credentials)
+* [Shared credentials file](/solutions/security/cloud/asset-disc-aws.md#cad-aws-use-a-shared-credentials-file)
+* [IAM role Amazon Resource Name (ARN)](/solutions/security/cloud/asset-disc-aws.md#cad-aws-use-iam-arn)
+
+::::{important}
+Whichever method you use to authenticate, make sure AWS’s built-in [`SecurityAudit` IAM policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor) is attached.
+::::
+
+
+
+#### Option 1 - Default instance role [cad-aws-use-instance-role]
+
+::::{note}
+If you are deploying to an AWS organization instead of an AWS account, you should already have [created a new role](/solutions/security/cloud/asset-disc-aws.md#cad-aws-setup-organization-manual), `cloudbeat-root`. Skip to step 2 "Attach your new IAM role to an EC2 instance", and attach this role. You can use either an existing or new EC2 instance.
```

Comment:
```suggestion
If you are deploying to an AWS organization instead of an AWS account, you should already have [created a new role](/solutions/security/cloud/asset-disc-aws.md#cad-aws-setup-organization-manual), `cloudbeat-asset-inventory-root`. Skip to step 2 "Attach your new IAM role to an EC2 instance", and attach this role. You can use either an existing or new EC2 instance.
```